Backpack.podspec

```diff
@@ -43,5 +43,5 @@ Pod::Spec.new do |s|
   s.dependency 'MBProgressHUD', '~> 0.9.1'
   s.frameworks = 'UIKit', 'Foundation', 'CoreText'
   s.requires_arc = true
-  s.swift_version = '4.0'
+  s.swift_version = '5.0'
```

Comment:
We can and should provide an array here `swift_versions = ['5.0', '4.2', '4.0']`

Reply:
ohh I see, thanks for info 👍

Reply:
I think this should be **versions** with an `s` per the [docs](https://guides.cocoapods.org/syntax/podspec.html#swift_versions)

Reply:
My change is `s.swift_versions` [here](https://github.com/Skyscanner/backpack-ios/pull/757/files#diff-ec850618833e8b72d1203858fa50fde5R46)

Reply:
Ah, I didn't notice that the diff was outdated. My bad